tests/jballerina-unit-test/src/test/resources/test-src/natives/builtin-symbol-negative.bal

```diff
@@ -9,3 +9,10 @@ type 'int int;
 type 'never never;
 type 'readonly readonly;
 type 'xml xml;
+
+public enum Foo {
```

Comment:
Shall we add a positive test to `org.ballerinalang.test.variable.shadowing.ShadowingTest` also?

e.g.,
```ballerina
type Foo int;

public function main() {
    string Foo = "hello";
}
```

Reply:
sure